content/app/guides/multi-app-solution/instructions/_index.en.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Instructions for making a multi-app solution in Altinn
+linktitle: Instructions
+description: Explanations of how to go about when creating a general multi-app solution
+weight: 20
+aliases:
+
+- /app/multi-app-solution/instructions/
+
+---
+
+## Prerequisites
+
+Before continue reading the technical implementation details, please make sure the
+required prerequisites are met.
+
+### Functional Prerequisites
+
+1. The app-developer must have access to create, develop and deploy applications owned by an organization.
+2. Existing basic Altinn form where the content of the data model is assessed against the intended data flow between
+   application A and application B.
+3. It should be clear who the instance owners are, meaning what roles and access they acquire.
+
+Does the end user of application A have the required roles or delegated accesses to instantiate application B on behalf
+of the intended
+recipient?
+
+- If yes, no maskinporten integration is required.
```

Comment:
This does not tell the user much. Is what we want to communicate here that if yes, skip technical prerequisites? if so let's state that instead. If not, let's reference the concrete parts of the documentation that does not apply to them.

Reply:
Agree. Maybe we can simply remove all the lines L24-L30? I guess the *-sentence gives the reader the same information? 🤔

Reply:
yup, let's remove this section.